app/mailers/application_mailer.rb

```diff
@@ -1,4 +1,4 @@
 class ApplicationMailer < ActionMailer::Base
-  default from: 'from@example.com'
+  default from: ENV['THESIS_ADMIN_EMAIL']
```

Comment:
Do we want to test for the presence of this key and have some kind of default (libdev-cs?) Or is it actually desirable for something to crash if this is unset?

Reply:
I'd lean towards a runtime error if the app isn't configured usefully myself, but I'd listen to a different argument.

Reply:
👍 , just want to make sure this is a specific choice (and also that we have documented that this must always be set)